content/java/concepts/strings/terms/getChars.md

```diff
@@ -0,0 +1,59 @@
+---
+Title: '.getChars()'
+Description: ' A java strings method that copies characters from the given string into the destination character array.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The **.getChars()** string method copies a set of characters from an array. The method then copies the characters into character array.
+
+## **Synthax**
+
+```pseudo
+str.getChars(12, 25, destArray, 0);
```

Comment:
You need to label the parameters of the method and describe them below. Use named variables rather than arbitrary numbers so you can describe their use below.